crates/noirc_evaluator/src/ssa_refactor/ir/basic_block.rs

```diff
@@ -35,7 +35,24 @@ pub(crate) struct BasicBlock {
 pub(crate) type BasicBlockId = Id<BasicBlock>;
 
 impl BasicBlock {
-    pub(super) fn new(parameters: Vec<ValueId>) -> Self {
+    pub(crate) fn new(parameters: Vec<ValueId>) -> Self {
         Self { parameters, instructions: Vec::new(), is_sealed: false, terminator: None }
     }
+
+    pub(crate) fn parameters(&self) -> &[ValueId] {
```

Comment:
These are the parameters of the basic block. Blocks don't have arguments since the arguments only become known when a jump instruction is created with the relevant arguments, which will correspond to the block's parameters here.

Note that a function's parameters are represented as the parameters of its entry block.